sdk/ai/azure-ai-client/azure/ai/client/models/__init__.py

```diff
@@ -21,16 +168,162 @@
+    "FunctionName",
+    "FunctionToolDefinition",
     "InputData",
+    "MessageAttachment",
+    "MessageContent",
+    "MessageDelta",
+    "MessageDeltaChunk",
+    "MessageDeltaContent",
+    "MessageDeltaImageFileContent",
+    "MessageDeltaImageFileContentObject",
+    "MessageDeltaTextAnnotation",
+    "MessageDeltaTextContent",
+    "MessageDeltaTextContentObject",
+    "MessageDeltaTextFileCitationAnnotation",
+    "MessageDeltaTextFileCitationAnnotationObject",
+    "MessageDeltaTextFilePathAnnotation",
+    "MessageDeltaTextFilePathAnnotationObject",
+    "MessageImageFileContent",
+    "MessageImageFileDetails",
+    "MessageIncompleteDetails",
+    "MessageTextAnnotation",
+    "MessageTextContent",
+    "MessageTextDetails",
+    "MessageTextFileCitationAnnotation",
+    "MessageTextFileCitationDetails",
+    "MessageTextFilePathAnnotation",
+    "MessageTextFilePathDetails",
+    "OpenAIFile",
+    "OpenAIPageableListOfAgent",
+    "OpenAIPageableListOfRunStep",
+    "OpenAIPageableListOfThreadMessage",
+    "OpenAIPageableListOfThreadRun",
+    "OpenAIPageableListOfVectorStore",
+    "OpenAIPageableListOfVectorStoreFile",
+    "RequiredAction",
+    "RequiredFunctionToolCall",
+    "RequiredFunctionToolCallDetails",
+    "RequiredToolCall",
+    "RunCompletionUsage",
+    "RunError",
+    "RunStep",
+    "RunStepCodeInterpreterImageOutput",
+    "RunStepCodeInterpreterImageReference",
+    "RunStepCodeInterpreterLogOutput",
+    "RunStepCodeInterpreterToolCall",
+    "RunStepCodeInterpreterToolCallDetails",
+    "RunStepCodeInterpreterToolCallOutput",
+    "RunStepCompletionUsage",
+    "RunStepDelta",
+    "RunStepDeltaChunk",
+    "RunStepDeltaCodeInterpreterDetailItemObject",
+    "RunStepDeltaCodeInterpreterImageOutput",
+    "RunStepDeltaCodeInterpreterImageOutputObject",
+    "RunStepDeltaCodeInterpreterLogOutput",
+    "RunStepDeltaCodeInterpreterOutput",
+    "RunStepDeltaCodeInterpreterToolCall",
+    "RunStepDeltaDetail",
+    "RunStepDeltaFileSearchToolCall",
+    "RunStepDeltaFunction",
+    "RunStepDeltaFunctionToolCall",
+    "RunStepDeltaMessageCreation",
+    "RunStepDeltaMessageCreationObject",
+    "RunStepDeltaToolCall",
+    "RunStepDeltaToolCallObject",
+    "RunStepDetails",
+    "RunStepError",
+    "RunStepFileSearchToolCall",
+    "RunStepFunctionToolCall",
+    "RunStepFunctionToolCallDetails",
+    "RunStepMessageCreationDetails",
+    "RunStepMessageCreationReference",
+    "RunStepToolCall",
+    "RunStepToolCallDetails",
+    "SubmitToolOutputsAction",
+    "SubmitToolOutputsDetails",
     "SystemData",
+    "ThreadDeletionStatus",
+    "ThreadMessage",
+    "ThreadMessageOptions",
+    "ThreadRun",
+    "ToolDefinition",
+    "ToolOutput",
+    "ToolResources",
+    "TruncationObject",
+    "UpdateCodeInterpreterToolResourceOptions",
     "UpdateEvaluationRequest",
-    "EndpointType", 
-    "AuthenticationType",
-    "CredentialsSASAuth",
+    "UpdateFileSearchToolResourceOptions",
```

Comment:
Please don't remove these three:
    "EndpointType", 
    "AuthenticationType",
    "CredentialsSASAuth"
I still need to figure out how to make the emitter include those. I add them manually for now.

Reply:
They are there, I put them alphabetical order